crates/bifrost/src/watchdog.rs

```diff
@@ -78,9 +101,10 @@ impl Watchdog {
 
     fn on_log_trim(&self, log_id: LogId, requested_trim_point: Lsn) {
         let bifrost = self.inner.clone();
+        let sender = self.trims_send.clone();
         let _ = TaskCenter::spawn_child(
             TaskKind::BifrostBackgroundLowPriority,
-            format!("trim-chain-{log_id}"),
+            format!("get-trim-point-{log_id}"),
```

Comment:
we could simplify things if the cc looked this up before passing it to the watchdog, but that would be even more stale and it could also slow down that cluster controller, so perhaps this is best.

we could also communicate this back to the main loop as a WatchDog command instead of a separate sender, but it feels a bit weird because this is internal only.